.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -187,18 +201,16 @@ jobs:
         fi
 
         if [[ "${{ matrix.os }}" == macos-* ]]; then
-          # Qt5_DIR set by Install Qt5 step
-          echo "cmake_prefix=$Qt5_DIR/lib/cmake;$Qt5_DIR" >> $GITHUB_OUTPUT
           brew install --quiet ccache autoconf automake libtool ninja && brew link --overwrite ccache
-          brew install libsvm xerces-c boost eigen sqlite coinutils cbc cgl clp
+          brew install libsvm xerces-c boost eigen sqlite coinutils cbc cgl clp qt@5
+          echo "cmake_prefix=$(brew --prefix qt@5)/lib/cmake;$(brew --prefix qt@5)" >> $GITHUB_OUTPUT
+          echo "Qt5_DIR=$(ls $(brew --prefix qt@5)/*)/lib/cmake/Qt5" >> $GITHUB_ENV
```

Comment:
Is this supposed to go to GITHUB_OUTPUT or GITHUB_ENV? ENV is currently choking on it.